### PR TITLE
Add upload_timeout to PictrsConfig

### PR DESCRIPTION
**Pull request by ismailkarsli**
Current timeout is not suitable for large files. Being able to customize it would be cool.

### CODE REVIEW

**Comment by ismailkarsli** [comment]
image

**Review by Nutomic — APPROVED** [review]

File: crates/utils/src/settings/structs.rs [hunk 1]
```diff
@@ -89,6 +89,10 @@ pub struct PictrsConfig {
   /// disabled.
   #[default(true)]
   pub cache_external_link_previews: bool,
+
+  /// Timeout for uploading images to pictrs
```
Comment (dessalines) [inline crates/utils/src/settings/structs.rs:93]: I'd add the units for this, seconds.

Comment (ismailkarsli) [reply crates/utils/src/settings/structs.rs:93]: You mean as a comment?

**Review by dessalines — APPROVED** [review]
Sweet, thx.

**Pull request merged**